std/mem.zig

```diff
@@ -248,25 +248,25 @@ pub fn copyBackwards(comptime T: type, dest: []T, source: []const T) void {
     }
 }
 
-pub fn set(comptime T: type, dest: []T, value: T) void {
+pub fn set(dest: var, value: @typeOf(dest[0])) void {
```

Comment:
Here as well.

Reply:
Looking at the docs, should this be `Pointer.child` or `Array.child`?

Reply:
`dest` is a slice (a type of pointer), not an array.